File: AngularProject/src/app/Classes/OrderPlace.ts
import { Time } from "@angular/common";

export class OrderPlace{
    constructor(public idOrder?:number,public idUser?:number,public orderDate?:Date, public orderTime?:Time ,public idTrip?:number,
        public numberPlaces?:number,public name?:string ,public targetTripe?:string ,public date ?:Date)
    {

    }
}

File: projectTrips/DTO/classes/OrderPlace.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO.classes
{
    public partial class OrderPlace
    {
        public short IdOrder { get; set; }

        public short? IdUser { get; set; }

        public DateTime? OrderDate { get; set; }

        public TimeSpan? OrderTime { get; set; }

        public short? IdTrip { get; set; }

        public short? NumberPlaces { get; set; }

        //הוספות שלי למסד!!!!!!!!!!!!!!!
        //בנוסף לקוד משתמש
        public string? Name { get; set; }
        //בנוסף לקוד הטיול
        public string? TargetTripe { get; set; }
        //תאריך
        public DateTime? Date { get; set; }
        //public virtual TheTrip? IdTripNavigation { get; set; }

        //public virtual User? IdUserNavigation { get; set; }
    }
}
